packages/contracts/src/dollar/mocks/MockShareV1.sol

```diff
@@ -2,8 +2,18 @@
 pragma solidity 0.8.19;
 
 import "../core/ERC1155Ubiquity.sol";
+import {Initializable} from "@openzeppelinUpgradeable/contracts/proxy/utils/Initializable.sol";
 
-contract BondingShare is ERC1155Ubiquity {
+contract BondingShare is Initializable, ERC1155Ubiquity {
     // solhint-disable-next-line no-empty-blocks
-    constructor(address _diamond) ERC1155Ubiquity(_diamond, "URI") {}
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize(
```

Comment:
i think this one could be an override from the original implementation?

Reply:
MockShareV1 is no longer used anywhere as I deploy with ERC1155Ubq directly in the test but can you elaborate on what you mean?